### ISSUE TITLE
Dexie Cloud Push Only Mode

### ISSUE THREAD

**Issue by flipdav8** [dexie/Dexie.js#1622]
Is there a way to set db.cloud.configure() to only push to the cloud database? 

My use case is a user logging in and submitting a form. I don't want each user to be able to read the other entries in the form table submitted by other users. Is write-only access possible at the moment?  

I have really enjoyed using Dexie Cloud, much appreciated!

**Comment by dfahlander** [comment]
Hi! Thanks for the feedback! Currently there's no write-only access available, but if the forms stored as "private" objects (which is the default when not setting `realmId`) they will only be visible / synced for the same user. But probably you need a way to aggregate the submitted forms somehow? Or put them in a realm where the user has write-only access but no read access. Let me know more about your use case and I'll see if it can be possible with current version or if it would be a possible feature / improvement!

**Comment by flipdav8** [comment]
Exactly, it would be great to aggregate the forms but only for a privileged user. My use case would be a survey-like tool where private information such as a contact number can be entered but without risk of other users accessing it. 

I've also setup encryption as suggested in #1604 using an environment variable as the encryption key so data is encrypted in transit. Combining this with write-only would be really handy :)

I have tried logging in as a demo user, setting the realmId to a known shared realmId, and then setting owner to null like below. I was hoping this way the user who owns/manages the realm could access the data but other demo users couldn't. I now realise the demo user needs to be a member of the realm for this to sync which defeats the goal. 

db.forms.add(
{
          data: "confidential data",
          realmId: *Known user realmId*
          owner: null,
})

**Comment by dfahlander** [comment]
Thanks for a good feature request. I'm currently looking in to what would be possible to do with the access control model to allow write-only access within a realm. I will post progress on this issue.

**Comment by dusty-phillips** [comment]
I am encountering a similar use case for read permissions, except the receiver of the feedback is not the system owner: 

User A wants to share an object with B, C, and D. Those other users can read but not modify the object in that table. They can post a response with add permissions in a different table. User A can read all responses, but B, C, and D should not be able to read each others’ responses.

My current solution is to create a separate realm for (B, A, obj, responseB), (C, A, obj, ResponseC), and (D, A, obj, responseD), but this means A has to store and sync multiple copies of obj, so it’s not ideal.
